kettle/src/main/resources/standalone/configuration/hawkular-realm.json

```diff
@@ -57,17 +57,19 @@
       "secret" : "${uuid.hawkular.accounts.backend}"
     },
     {
-      "name": "hawkular-accounts-ui",
+      "name": "hawkular-ui",
       "enabled": true,
       "bearerOnly" : false,
       "publicClient": true,
-      "secret": "${uuid.hawkular.accounts.ui}",
+      "secret": "${uuid.hawkular.ui}",
       "fullScopeAllowed" : false,
       "redirectUris": [
+        "http://localhost:8080/*",
```

Comment:
If the port is hard coded it doesn't work with docker when mapping the port 8080 to a different port. That's actually a suggested way to run it (see http://www.hawkular.org/docs/user/getting-started.html).

@jpkrohling can't be the port automagically inferred somehow? I remember I had similar issue with GWT debug mode and KeyCloak. (GWT debug mode was running on different port than production app)